src/defaultapps/terminal/commands.js

```diff
@@ -0,0 +1,15 @@
+var tree = (res) => {
+    var element = document.createElement("div");
+    element.appendChild(document.createTextNode('user@root:~$ tree'))
+    for(let i=0;i<4;i++){
+        var terminalOutput=document.getElementById('terminalOutput');
```

Comment:
For now this is ok but contents should be fetched from the FS we will be creating later.

Reply:
i had created a `fs.json` but since it is local, we just have to give its `path` to the `tree()`.
I used a loop of 4 since there are only 4 elements right now in the json. I will change this loop later :)